lib/ButtonNew/ButtonIcon/ButtonIcon.js

```diff
@@ -19,48 +19,28 @@ function ButtonIcon({
   iconTitle,
   children,
   enabled,
+  contained,
```

Comment:
🚩 Do we still need this? I can't see it being used in the component apart from proptypes 👁️